buildSrc/src/main/groovy/Classpaths.groovy

```diff
@@ -287,4 +287,31 @@ class Classpaths {
         Configuration config = p.configurations.getByName(configName)
         addDependency(config, GUAVA_GROUP, GUAVA_NAME, GUAVA_VERSION)
     }
+
+    static void inheritParquetHadoop(Project p, String configName = JavaPlugin.IMPLEMENTATION_CONFIGURATION_NAME) {
+        Configuration config = p.configurations.getByName(configName)
+        addDependency(config, 'org.apache.parquet', 'parquet-hadoop', '1.13.0')
```

Comment:
I was expecting a version bump for `parquet-hadoop`, and maybe for its dependencies. What version have we identified as "fixing" the problem we addressed with our forked file?

Reply:
I can check with Andy for which version has the bug fix.
I saw that Andy was using 1.12.3 on the enterprise side and our code had already been upgraded to 1.13.0. So didn't need to make any version changes.

Reply:
Let's file a separate issue to check on versions and bump these if applicable. No need to do it as part of this PR.

Reply:
Created #4491